### PR TITLE
Added Inventory App

### PR DESCRIPTION
**Pull request by aryendersingh**
Updated the inventory app with the order attributes!

### CODE REVIEW

File: inventory/Inventory.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Inventory:
+    def __init__(self, item_id,item_name,current_level, target_level, price,isOrdered, refillNeeded):
+        self.item_id = item_id
+        self.item_name = item_name
+        self.current_level = current_level
+        self.target_level = target_level
+        self.price = price
+        self.isOrdered = isOrdered
+        self.refillNeeded = refillNeeded
+
```
Comment (khusbupatel) [inline inventory/Inventory.py:10]: delete isOrdered

File: inventory/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        migrations.CreateModel(
+            name='Inventory',
+            fields=[
+                ('item_id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('item_name', models.CharField(max_length=200)),
+                ('current_level', models.IntegerField(default=0)),
+                ('target_level', models.IntegerField(default=0)),
+                ('price', models.FloatField(default=0.0)),
+                ('refillNeeded', models.BooleanField(default=False)),
+                ('isOrdered', models.BooleanField(default=False))
```
Comment (khusbupatel) [inline inventory/migrations/0001_initial.py:23]: snake case

File: inventory/models.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# Create your models here.
+
+class Inventory(models.Model):
+    item_id = models.IntegerField(default=0,unique=True,primary_key=True)
+    #non-null name
+    item_name = models.CharField(max_length = 200)
+    current_level = models.IntegerField(default=0)
+    target_level = models.IntegerField(default=0)
+    price = models.FloatField(default=0.0)
```
Comment (khusbupatel) [inline inventory/models.py:12]: delete price

File: inventory/views.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
+from django.shortcuts import render
+import json
+from .models import Inventory
+from .serializers import InventorySerializer
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+# Create your views here.
+
```
Comment (khusbupatel) [inline inventory/views.py:9]: add get all items

File: inventory/views.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        return Response(status=status.HTTP_500_BAD_REQUEST)
+
+@api_view(['POST'])
+def add_Item(request):
+    serializer = InventorySerializer(data = request.data)
+    if serializer.is_valid():
+        serializer.save()
+        return Response(serializer.data)
+    return Response(serializer.errors)
+
```
Comment (khusbupatel) [inline inventory/views.py:29]: this function may not be used

File: inventory/views.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+@api_view(['GET'])
+def get_Item(request):
+    body_unicode = request.body.decode('utf-8')
+    try:
+        body_data = json.loads(body_unicode)
+        item_id = body_data["item_id"]
+        item = Inventory.objects.get(id = item_id)
+        serializer = InventorySerializer(item)
+        return Response(serializer.data)
```
Comment (khusbupatel) [inline inventory/views.py:39]: add status code

File: inventory/views.py [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+            Inventory.objects.filter(pk = item_id).update(target_level=new_target_level)
+        if ("price" in body_data.keys()):
+            new_price = body_data["price"]
+            Inventory.objects.filter(pk = item_id).update(price=new_price)
+        if ("isOrdered" in body_data.keys()):
+            new_isOrdered = body_data["isOrdered"]
+            Inventory.objects.filter(pk = item_id).update(isOrdered=new_isOrdered)
+        if ("refillNeeded" in body_data.keys()):
+            new_refillNeeded = body_data["refillNeeded"]
+            Inventory.objects.filter(pk = item_id).update(refillNeeded=new_refillNeeded)
```
Comment (khusbupatel) [inline inventory/views.py:70]: do not need this function

**Comment by aryendersingh** [comment]
Not required anymore

**Pull request closed without merging**